src/Microsoft.AspNet.Mvc.Core/Logging/ScopeValues.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.Framework.Internal;
+using Microsoft.Framework.Logging;
+
+namespace Microsoft.AspNet.Mvc.Logging
+{
+    public class ScopeValues : LoggerStructureBase
+    {
+        public ScopeValues([NotNull] string scopeName)
+        {
+            ScopeName = scopeName;
+        }
+
+        public string ScopeName { get; }
```

Comment:
FYI..I am doing this on purpose so that this property name will end up as a field in elastic search and would be easy to search.